framework/core/buffer.cpp

```diff
@@ -38,10 +38,12 @@ Buffer::Buffer(Device &device, VkDeviceSize size, VkBufferUsageFlags buffer_usag
 	memory_info.usage = memory_usage;
 	memory_info.flags = VMA_ALLOCATION_CREATE_MAPPED_BIT;
 
+    VmaAllocationInfo allocInfo{};
```

Comment:
We don't have at the moment a CI to flag coding style.

We would like all the PR to use the `snake_case` style for naming variables and functions.